games/minetest/Portfile

```diff
@@ -44,27 +45,44 @@ post-extract {
     }
 }
 
-license                 LGPL-2.1+
+license                 LGPL
 categories              games
 platforms               darwin
-maintainers             nomaintainer
+maintainers             @Zweihorn
 description             open source infinite-world block sandbox game with survival and crafting
-long_description        ${description}.
-
-depends_build-append    path:bin/doxygen:doxygen \
-                        port:mesa
+long_description        - ${description} - \
+                         \
+                        Minetest (usually abbreviated as MT) is an open source voxel game \
+                        engine. Play one of our many games, mod a game to your liking, make your \
+                        own game, or play on a multiplayer server. \
+                         \
+                        Available for Windows, macOS, GNU/Linux, FreeBSD, OpenBSD, DragonFly \
+                        BSD, and Android. \
+                         \
+                        ** Minetest at <https://www.minetest.net> \
+                        * MT mods at <https://content.minetest.net/> * \
+                        * Database explained at <https://wiki.minetest.net/Database_backends> ** \
+                         \
+                        This ${name} port provides a cutting edge Minetest (stable).
+
+homepage                https://www.minetest.net
+
+depends_build-append    port:doxygen
 
 depends_lib-append      port:irrlichtmt \
-                        path:include/turbojpeg.h:libjpeg-turbo \
+                        port:libjpeg-turbo \
                         port:libogg \
                         port:libvorbis \
                         port:freetype \
                         port:gettext \
                         port:leveldb \
-                        port:hiredis \
+                        port:sqlite3 \
+                        port:zstd \
+                        port:luajit \
```

Comment:
revert to the original with path-style dependency

Reply:
NO - will be removed if no other reasons show some proof here

This is only consistent with removal of the redundant REDIS which was built but not used by the leagy Profile.

see at: https://github.com/macports/macports-ports/pull/17077#discussion_r1058812083